db/migrate/20170109190411_make_job_belong_to_miq_task.rb

```diff
@@ -0,0 +1,27 @@
+class MakeJobBelongToMiqTask < ActiveRecord::Migration[5.0]
+  class Job < ActiveRecord::Base;
+    belongs_to :miq_task
+  end
+  class MiqTask < ActiveRecord::Base; end
+
+  def change
+    add_reference :jobs, :miq_task, type: "bigint", index: true
+    reversible do |direction|
```

Comment:
I agree, do the schema change in one migration, data change in another.

I like `change` for simple things but prefer up and down when they're clearly doing different things.